src/Client.js

```diff
@@ -172,7 +172,7 @@ class Client extends EventEmitter {
             }
         );
 
-        const INTRO_IMG_SELECTOR = '[data-icon=\'chat\']';
```

Comment:
Should be changed to `search` right? @elhumbertoz (Better query)

Reply:
![image](https://github.com/pedroslopez/whatsapp-web.js/assets/45042245/b339d0db-bedb-4b61-b70d-41ef097c6753)